SpotifyInterface.c

```diff
@@ -346,15 +346,21 @@ bool Spotify_SendCommand(SpotifyInterfaceHandler_t SpotifyInterfaceHandler, int
             break;
 
         case GetUserInfo:
+        //SEZ@Done ESP_LOGW just for test.
+        ESP_LOGW(TAG, "GetUserInfo");
             Spotify_GetInfo(Command, PrivateHandler.token.AccessToken);
-            IsSuccessfull = PrivateHandler.SpotifyBuffer.status == 200;
+        ESP_LOGW(TAG, "Spotify_GetInfo");    
+            IsSuccessfull = (PrivateHandler.SpotifyBuffer.status == 200)? true:false;
             if (!IsSuccessfull)
             {
                 ESP_LOGW(TAG, "No user is found");
                 retValue = false;
                 break;
             }
-            ExtractUserInfoParamsfromJson(PrivateHandler.SpotifyBuffer.MessageBuffer, InterfaceHandler->UserInfo);
+        ESP_LOGW(TAG, "IsSuccessfull");    
+            ExtractUserInfoParamsfromJson(PrivateHandler.SpotifyBuffer.MessageBuffer, 
+            InterfaceHandler->UserInfo);
+            ESP_LOGW(TAG, "ExtractUserInfoParamsfromJson");    
```

Comment:
and same here. if the status is going to be reported later, this one is not needed. Maybe they could be `ESP_LOGI` or `ESP_LOGD` to be visibale only during debug